front/lib/api/assistant/generation.ts

```diff
@@ -320,6 +320,13 @@ export async function constructPromptMultiActions(
   context += `local_time: ${d.format("YYYY-MM-DD HH:mm (ddd)")}\n`;
   if (owner) {
     context += `workspace: ${owner.name}\n`;
+    context += `user_username: ${userMessage.context.username}\n`;
+    if (userMessage.context.fullName) {
+      context += `user_full_name: ${userMessage.context.fullName}\n`;
+    }
```

Comment:
Why not sending just the Full name? This is the one that was potentially edited by the user on signup.

Reply:
Fair removing username